mailheader.js

```diff
@@ -135,7 +135,12 @@ function _decode_rfc2231 (params) {
 
         params.cur_key = key_actual;
         params.keys[key_actual] = '';
-        value = decodeURIComponent(value);
+        try {
+            value = decodeURIComponent(value);
+        }
+        catch (e) {
+            logger.logerror("Decode header failed: " + key + ": " + value);
```

Comment:
I get the feeling it won't help much. Knowing the actual key/value is more beneficial.